astropy/coordinates/tests/test_angles.py

```diff
@@ -554,26 +554,89 @@ def test_angle_string():
     with pytest.warns(IllegalSecondWarning):
         a = Angle('00:00:60', u.deg)
     assert str(a) == '0d01m00s'
+    a = Angle('00:00:59S', u.deg)
+    assert str(a) == '-0d00m59s'
+    a = Angle('00:00:59N', u.deg)
+    assert str(a) == '0d00m59s'
+    a = Angle('00:00:59E', u.deg)
+    assert str(a) == '0d00m59s'
+    a = Angle('00:00:59W', u.deg)
+    assert str(a) == '-0d00m59s'
     a = Angle('-00:00:10', u.hour)
     assert str(a) == '-0h00m10s'
+    a = Angle('00:00:59S', u.hour)
+    assert str(a) == '-0h00m59s'
+    a = Angle('00:00:59N', u.hour)
```

Comment:
I know we cannot stop this, but I would still prefer to not test something that is wrong, so I suggest to remove this one.